guides/release/templates/input-helpers.md

```diff
@@ -52,7 +52,7 @@ To dispatch an action on specific events such as `key-press`, use the following
 
 ```handlebars
 <label for="firstname">First Name</label>
-{{input id="firstname" value=this.firstName key-press=(action "updateFirstName")}}
+<Input @id="firstname" @value={{this.firstName}} @key-press={{action "updateFirstName"}} />
```

Comment:
@locks we're recommending `key-press` here and `keypress` in the next section down. Should we land on a standard approach? I don't see that this should block this PR, but might be worth a follow up PR to clean up?